.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest, windows-latest, macos-latest]
-        php-versions: ['5.6', '7.4', '8.0', '8.1', '8.2', '8.3']
+        php-versions: ['5.6', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4']
```

Comment:
This is a good question. 

I would say: As long as we don't have a clear strategy which version to support, I would keep all versions.

Personally, I would leave out all versions up to 8.0 and only support the current versions 8.1-8.4. This would also allow us to modernize the code a bit. But we should discuss in another thread what we really try to achieve with this test library. The unique selling points I see are that Enhance PHP is delivered in one file, has no dependencies and is very easy to use.

Reply:
Okay, let's start a thread on support for versions.

Those three concepts you mention are the general idea. We didn't want folks to have to import lots of stuff to be able to test - it should be easy to just add a file and start coding, and not have to make additional decisions to select mocking and assertion utilities.